src/settings/MappingProfiles/MappingProfilesForm.js

```diff
@@ -162,6 +162,22 @@ export const MappingProfilesFormComponent = ({
       onLink: setAddedRelations,
       onUnlink: setDeletedRelations,
     },
+    'item-status': {
+      acceptedValuesList: [
+        { name: 'Available' },
```

Comment:
Since the values are shown on the UI it means they won't be localized. Does it make sense to create a key-value pair for having the keys (keys can be stored in constants) for coding and localization?
```
{
 key: 'AWAITING_PICKUP',
 value: <FormattedMessage ...
}
```

Reply:
Done